setup.py

```diff
@@ -48,7 +48,7 @@
 # Prepare install requires and extra requires
 install_requires = [
     'fmf>=1.2.1',
-    'click',
+    'click>=8, <8.1.4',
```

Comment:
Perhaps, shall we include a `TODO` comment here pointing to the `click` issue so that it's clear why the version freezed and we don't forget to remove it later?

Reply:
This one? https://github.com/teemtee/tmt/pull/2199/commits/297a4d1b8f94d04c8819d4c33a642235f9568ab1

Reply:
Yeap, thanks!